wsi/drm/drmbuffer.cpp

```diff
@@ -337,20 +327,20 @@ const MediaResourceHandle& DrmBuffer::GetMediaResource(MediaDisplay display,
 
   VASurfaceAttribExternalBuffers external;
   memset(&external, 0, sizeof(external));
-  uint32_t rt_format = DrmFormatToRTFormat(format_);
+  uint32_t rt_format = DrmFormatToRTFormat(image_.handle_->meta_data_.format_);
```

Comment:
should be just format_ here and all other places which used format_ ? or wondering if we can store format value directly in meta_data_. did u check that and saw any failures ?

Reply:
You're right, this should be format_, I didn't see any failures when I tested this.  I'll check to see if format_ is being referenced elsewhere just in case.